dev-libs/libbson/libbson-1.19.1.ebuild

```diff
@@ -0,0 +1,48 @@
+# Copyright 1999-2021 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+inherit cmake
+
+DESCRIPTION="Library routines related to building,parsing and iterating BSON documents"
+HOMEPAGE="https://github.com/mongodb/mongo-c-driver/tree/master/src/libbson"
+SRC_URI="https://github.com/mongodb/mongo-c-driver/releases/download/${PV}/mongo-c-driver-${PV}.tar.gz -> ${P}.tar.gz"
+
+LICENSE="Apache-2.0"
+SLOT="0"
+KEYWORDS="amd64 ~hppa ~ppc ~riscv ~sparc x86"
+IUSE="examples static-libs"
+
+DEPEND="dev-python/sphinx
+	>=dev-python/docutils-0.14
+	<dev-python/docutils-0.18"
```

Comment:
I am absolutely not sure about that but the build breaks with this error:
```
pkg_resources.DistributionNotFound: The 'docutils<0.18,>=0.14' distribution was not found and is required by Sphinx
```
when installing with docutils-0.18.1.